src/App.css

```diff
@@ -10,6 +10,7 @@ body {
 h1 {
   font-weight: normal;
   font-size: calc(24px + 1vw);
+  line-height: 29px;
```

Comment:
After Ajara's suggestion on making the site title to be h1, this might affect other pages, could you merge master into your branch and see if this is working as intended for the whole site?

Reply:
Yes I can. If anything, I can also add it as a separate class to ensure when the text wraps, it has line space.

Reply:
Yes it is.